src/main/perl/TextRender.pm

```diff
@@ -154,16 +153,10 @@ Java properties format (using C<Config::Properties>),
 
 .INI format (using C<Config::Tiny>)
 
-=item general
-
-(using C<Config::General>).
-
-Caution: there is no way to garantee reproducible results as
-there is no control over the order of hash keys. It is recommended to
-try to generate this format in another way (e.g. via TT).
-
 =back
 
+(Builtin module <general> is provided as part of L<CCM::TextRender> since 15.10).
```

Comment:
Thanks for the detailed explanation. This area is much more complex than I had remembered - the link to the blog post was a handy refresher. I agree with the formulation you propose.